core/src/test/java/kafka/server/share/SharePartitionManagerTest.java

```diff
@@ -1068,10 +1070,10 @@ public void testProcessFetchResponse() {
         Map<SharePartitionManager.SharePartitionKey, SharePartition> partitionCacheMap = new ConcurrentHashMap<>();
         partitionCacheMap.computeIfAbsent(new SharePartitionManager.SharePartitionKey(groupId, tp0),
             k -> new SharePartition(groupId, tp0, MAX_IN_FLIGHT_MESSAGES, MAX_DELIVERY_COUNT,
-                RECORD_LOCK_DURATION_MS, mockTimer, new MockTime(), NoOpShareStatePersister.getInstance()));
+                    mockTimer, new MockTime(), NoOpShareStatePersister.getInstance(), DEFAULT_RECORD_LOCK_DURATION_MS, GROUP_CONFIG_MANAGER));
```

Comment:
indentation issue
might want to set correct value in idea

settings > editor > code style > java > set tab/indent/continuation ident to 5

There are some more occurrences below